_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -6,6 +6,10 @@ This section will be updated prior to the next release.
 
 ### Upgrade warnings
 
+#### New kind of facts in Discovery plugin
```

Comment:
Is this in nightly (2.6/3.0 to be) or in 2.5? If it's 2.5 then you need to put it in the correct file.

Reply:
Good point! I will update the PR after relase notes will be merged. This PR with new feature is already merged so that means it should be in 2.5 notes, right?

Reply:
Yes, if this is part of 2.5 then 2.5 the correct manual.

Reply:
What Ewoud said, this needs to be done in the 2.5 branch and not nightly.